src/model/position.js

```diff
@@ -140,7 +143,7 @@ export default class Position {
 	 * @type {Number}
 	 */
 	get offset() {
-		return last( this.path );
```

Comment:
Why this change? I thought myself that `last()` is probably slow, but I didn't see any change in results.

Reply:
I don't see a need here for using an external library which involves loading an external library for this. Also for other path operations, we do not use libs for simple tasks in `Position` elsewhere.

The `last` helper does:

```js
function last(array) {
  var length = array == null ? 0 : array.length;
  return length ? array[length - 1] : undefined;
}
```

and we should never have `Position` without `this.path`.

Calling external method will be slower. It is arguable if this is an important gain. Here I get 195.7ms (1.1%) down to 2.3 ms (0.0%). In a total of ~20s (long semantic + undo). So it is arguable if this change is necessary - I'm for this change.

Current:
![](https://user-images.githubusercontent.com/247363/79196530-9285e580-7e30-11ea-9257-07e7be6ec231.png)

Current with reverting to `last`.
![Selection_307](https://user-images.githubusercontent.com/247363/79196602-a9c4d300-7e30-11ea-84cb-6e203d0152ed.png)

TL;DR: Let's keep it - using `last()` makes no sense.